metriq_gym/run.py

```diff
@@ -89,11 +95,60 @@ def poll_job(args: argparse.Namespace, job_manager: JobManager) -> None:
     ]
     if all(task.status() == JobStatus.COMPLETED for task in quantum_job):
         result_data: list[ResultData] = [task.result().data for task in quantum_job]
-        print(handler.poll_handler(job_data, result_data))
+        results: BenchmarkResult = handler.poll_handler(job_data, result_data)
+        print(results)
+        if is_upload:
+            upload_job(args, job_type, job_data, results, platforms[metriq_job.device_name.lower()])
     else:
         logger.info("Job is not yet completed. Please try again later.")
 
 
+def upload_job(args: argparse.Namespace, job_type: JobType, job_data: BenchmarkData, result_data: BenchmarkResult, platform: int):
+    client = MetriqClient(args.token)
+    task = 0
+    method = 0
+    if job_type == JobType.QUANTUM_VOLUME:
+        task = 235 #Quantum Volume task ID
+        method = 144 #Heavy output generation task ID
+    elif job_type == JobType.BSEQ:
+        task = 236 #BSEQ task ID
+        method = 426 #BSEQ method ID
+    else:
+        raise Exception("You're trying to upload an unrecognized job type!")
+    client.submission_add_task(args.submission_id, task)
+    client.submission_add_method(args.submission_id, method)
+    client.submission_add_platform(args.submission_id, platform)
```

Comment:
Instead of getting the `platform` from `metriq_metadata` do we want to get this from the CLI metadata instead? This would solidify the elimination of the dependence of `metriq_metadata.py` as well so we could remove as an added bonus.

Reply:
I'd prefer not to, because `metriq_metadata.py` gets this right with much less potential for user error, and without making it the responsibility of the user to also look up the platform key.